### PR TITLE
user edit move to profile

### PR DESCRIPTION
**Pull request by marikrebega**
[Trello Task](https://trello.com/c/RehAcBh9/80-user-edit-move-to-profile)

**CHECK LIST**
- [x] specs added
- [x] specs passed
- [x] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [ ] I've checked new feature as logged in and logged out user if needed
- [x] "let!" is used in specs
- [x] screenshot of feature is added if task has UI

Знімок екрану з 2020-11-10 23-29-21

### CODE REVIEW

**Review by Andrii-Kozak — APPROVED** [review]

**Review by DenisRem — APPROVED** [review]

**Review by mmarusyk — APPROVED** [review]

**Review by matsevich — APPROVED** [review]

File: app/models/user.rb [hunk 1]
```diff
@@ -14,7 +14,9 @@ class User < ApplicationRecord
   mount_uploader :avatar, AvatarUploader
 
   belongs_to :osbb, optional: true
-  has_one :address, as: :addressable, dependent: :destroy
+  has_one :address, as: :addressable, dependent: :destroy, inverse_of: :addressable
+  validates_associated :address
```
[the remark this answers is not in the document]
Comment (marikrebega) [reply app/models/user.rb]: done

File: app/views/account/users/_address_fields.html.slim [hunk 2]
```diff
@@ -0,0 +1,18 @@
+fieldset
+  legend = t('.legend')
+  = f.simple_fields_for :address_attributes, @user.address do |address_form|
+    .form-group.user-form
+      = address_form.input :country, priority: [ @user.address&.country || 'Ukraine' ], label: t('.country'),
```
Comment (pereslop) [inline app/views/account/users/_address_fields.html.slim:5]: i18nt

File: spec/controllers/account/users_controller_spec.rb [hunk 3]
```diff
@@ -53,6 +55,34 @@
       end
     end
 
+    context "with valid address" do
+      let!(:valid_attributes_address) { valid_params[:address_attributes] = address_valid_params }
+
+      before do
+        put :update, params: { id: user.id,
+                               user: valid_params }
```
Comment (pereslop) [inline spec/controllers/account/users_controller_spec.rb:63]: you don't need to write request in two lines if is not too long

File: app/views/account/users/_address_fields.html.slim (hunk 2 above)
Comment (chu4) [reply app/views/account/users/_address_fields.html.slim:5]: на переклади вже забиваємо, бо наразі кожен з ним попрацював і спробував
і тепер лише час займає

File: config/locales/en.yml [hunk 4]
```diff
@@ -267,16 +275,7 @@ en:
           btn-cancel: 'Cancel'
         form-import:
           error-notice: 'prevented us from importing your spreadsheet. Please update your spreadsheet and try agin.'
-
```
Comment (chu4) [inline config/locales/en.yml:270]: why did you remove this?

Comment (marikrebega) [reply config/locales/en.yml:270]: @matsevich added a duplicate field

File: spec/controllers/account/users_controller_spec.rb (hunk 3 above)
Comment (marikrebega) [reply spec/controllers/account/users_controller_spec.rb]: done

**Review by chu4 — APPROVED** [review]
